lib/utils/data/globals.dart

```diff
@@ -78,20 +79,27 @@ double verticalPadding(BuildContext context, double size) =>
 
 List<String> timeZones = [];
 
-List<String> wallpapers = [
-  "images/wallpapers/dahliaOS_white_logo_pattern_wallpaper.png",
-  "images/wallpapers/dahliaOS_white_wallpaper.png",
-  "images/wallpapers/Gradient_logo_wallpaper.png",
-  "images/wallpapers/Three_Bubbles.png",
-  "images/wallpapers/Bubbles_wallpaper.png",
-  "images/wallpapers/Mountains_wallpaper.png",
-  "images/wallpapers/mountain.jpg",
-  "images/wallpapers/forest.jpg",
-  "images/wallpapers/modern.png",
-  "images/wallpapers/modern_dark.png",
-  "images/wallpapers/wood.jpg",
-  "images/wallpapers/beach.jpg",
-];
+Future<List<Wallpaper?>?> getWallpapers() async {
+  final response = await get(
+    Uri.parse(
+      'https://api.github.com/repos/dahliaOS/wallpapers/contents/Official/Desktop/PNG?ref=main',
```

Comment:
why png?
why not svg?

5.4 MB https://github.com/dahliaOS/wallpapers/blob/main/Official/Desktop/PNG/bubbles.png
24 KB https://github.com/dahliaOS/wallpapers/blob/main/Official/Desktop/SVG/bubbles.svg

Reply:
You are comparing the size of a rendered image and a non-rendered one, you do understand SVG images need to be rendered to be visually shown? And that's kind of an issue because Flutter itself does not support SVGs.

Reply:
> SVG images need to be rendered to be visually shown

space time tradeoff ...
svg2png rendering + caching can be done client side

> Flutter itself does not support SVGs

lol what?

Reply:
svgs on flutter are finnicky + the standard is a mess + they aren't exactly cheap to compute, the space tradeoff isn't worth for a wallpaper that takes more time to render

Reply:
> the space tradeoff isn't worth

yeah, in a perfect world i have infinite bandwidth + diskspace.
imagine a music player not supporting opus/mp3 codecs
cos "its hard to get right" and "just use wav" ...

at least there should be ~50 KB thumbnails, so i dont waste 5 MB per image

Reply:
this change really tries to add wallpaper repository support for pangolin, in another commit we should start reading the distro bundled wallpapers (so the used bandwidth comes from the iso itself and not pangolin itself). also caching should help this by a lot, both api rate limit and bandwidth (with maybe a manual way to refresh the cache if needed)